Dockerfile

```diff
@@ -130,6 +132,7 @@ RUN GIT_COMMIT=$GIT_COMMIT \
 # Final image
 ####################################################################################################
 FROM argocd-base
+ENTRYPOINT ["/usr/bin/tini", "--"]
```

Comment:
Entrypoint will not change, moving it here ensures he layer does not have to be rebuild due to the COPY layer and therefore also doesn't need to be pushed neither be pulled by consumers of the image.